### PR TITLE
Specify what you're doing

### PR DESCRIPTION
**Pull request by aandrieiev**

### CODE REVIEW

**Comment by rgabo** [comment]
@aandrieiev great feature idea. No test coverage though, can you please add tests?

**Comment by aandrieiev** [comment]
Frankly, I hope that the Ruby Core team members do take care of testing core functionality, like defining modules and internal methods. But just to make you happy, here you go!

**Comment by rgabo** [comment]
@aandrieiev love it :smile:

`We::Are::Hiring.you` was a public method so I thought test coverage was due. I will set up Travis CI once the PR is merged!

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,7 @@
 source 'https://rubygems.org'
 
+gem 'minitest'
```
Comment (rgabo) [inline Gemfile]: Dependencies of a gem should be defined in `we-are-hiring.gemspec` using `add_development_dependency`.

Comment (aandrieiev) [inline Gemfile]: This is not a dependency, it's part of the test suite and it shouldn't be distributed along with the gem =P

Comment (aandrieiev) [inline Gemfile]: Looks like I was wrong, `add_development_dependency` doesn't install/activate deps by default. FXD.

**Comment by rgabo** [comment]
@aandrieiev specifying Ruby 2.2 in the Gemfile should instruct Travis to use a recent Ruby and the PR should pass without a .travis.yml file :wink:

**Comment by rgabo** [comment]
maybe that's not a good idea though, we want everyone to be able to use this super gem

**Comment by aandrieiev** [comment]
I'd go with the actual .travis.yml file, it's just more explicit IMO. But anyway, do you really want to waste cycles on this? =D

**Comment by rgabo** [comment]
@aandrieiev check #2 :laughing:

**Comment by rgabo** [comment]
btw, we're already at 267 downloads, third most popular `hiring` gem: https://rubygems.org/search?query=hiring

**Comment by rgabo** [comment]
I think our 1.2.0 release will be a big hit

**Pull request merged**